vars/buildPlugin.groovy

```diff
@@ -279,10 +279,14 @@ def call(Map params = [:]) {
                    * plugins' PR builds could not be consumed by anything else anyway, and all
                    * plugins currently in the BOM are incrementalified.
                    */
-                  if (incrementals && currentBuild.currentResult == 'SUCCESS') {
+                  if (incrementals && currentBuild.currentResult == 'SUCCESS' && useContainerAgent) {
```

Comment:
This might be unnecessary, but I only tested in containers so I thought I should rather be safe than sorry. If the operators assure me that this is not needed, I can remove it, but I do not plan to do more thorough testing because commit collection is only needed on a best-effort basis anyway, so from a correctness point of view it is not a big deal if we are missing some commits. Note that I did test this in containers in both Linux and Windows.